jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/JKubeException.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2019 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at:
+ *
+ *     https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.jkube.kit.common;
+
+public class JKubeException extends RuntimeException {
+
+  public JKubeException(String message) {
+    super(message);
+  }
```

Comment:
I see this being used only in HelmService . We also seem to have JKubeServiceException . Why do we need a new exception class?